AutoRest/Generators/Extensions/Azure.Extensions/AzureExtensions.cs

```diff
@@ -86,8 +86,8 @@ public static void NormalizeAzureClientModel(ServiceClient serviceClient, Settin
             AddLongRunningOperations(serviceClient);
             AddAzureProperties(serviceClient);
             SetDefaultResponses(serviceClient);
-            AddParameterGroups(serviceClient);
             AddPageableMethod(serviceClient, codeNamer);
+            AddParameterGroups(serviceClient); //This should come after all methods have been dynamically added
```

Comment:
But this will create duplicate grouped parameter types. `AddPageableMethod` has special handling for grouped parameters - if they are in fact the same type the `Next` method will no create a new grouped parameter type.